validation/src/main/scala/hmda/validation/rules/ts/syntactical/S013.scala

```diff
@@ -1,11 +1,14 @@
 package hmda.validation.rules.ts.syntactical
 
 import hmda.model.fi.ts.TransmittalSheet
-import hmda.validation.dsl.{ CommonDsl, Result }
-
+import hmda.validation.dsl.Result
 import scala.concurrent.{ ExecutionContext, Future }
 
-object S013 extends CommonDsl {
+object S013 {
```

Comment:
The `apply` method for this edit has a different signature than the parent class. It is not clear what is going to happen with this edit, since it doesn't make much sense given how we are building the submission process.